### PR TITLE
Replace a bunch of ionicons with lucide

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

yes there's more

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

For example

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Tested most of them hehe

---

Resolves ABA-1009

### CODE REVIEW

File: packages/lego-bricks/src/components/Layout/Page/Page.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const Page = ({
         className={title ? styles.title : undefined}
       >
         {sidebar?.side === 'left' && <SidebarTrigger />}
-        {skeleton ? (
+        {skeleton && !title ? (
```
Comment (ivarnakken) [inline packages/lego-bricks/src/components/Layout/Page/Page.tsx:66]: no need to show skeleton if we have a title

Comment (eikhr) [reply packages/lego-bricks/src/components/Layout/Page/Page.tsx:66]: Eeeeeh, idk. I think we might get a lot of f.ex. "Editing: undefined" while loading.

Comment (ivarnakken) [reply packages/lego-bricks/src/components/Layout/Page/Page.tsx:66]: Yes that's why I also edited how the title on the event administrate pages are set.

However, I do believe it's a bad use of the `Page` component though if you manage to get "Editing: undefined". Would you agree to a hacky, but viable, solution like

```suggestion
        {skeleton && (!title || title.includes("undefined"))  ? (
```

Comment (ivarnakken) [reply packages/lego-bricks/src/components/Layout/Page/Page.tsx:66]: It's just annoying that the title starts "loading" when switching tabs

File: app/routes/events/components/EventAdministrate/Allergies.tsx [hunk 2]
```diff
@@ -30,7 +31,7 @@ export const canSeeAllergies = (
 const Allergies = () => {
   const { eventId } = useParams<{ eventId: string }>();
   const event = useAppSelector((state) =>
-    selectEventById(state, { eventId }),
+    eventId ? selectEventById(state, { eventId }) : undefined,
```
Comment (Bestem0r) [inline app/routes/events/components/EventAdministrate/Allergies.tsx:34]: Simplify?
```suggestion
    eventId && selectEventById(state, { eventId }),
```

File: app/routes/events/components/EventAdministrate/index.tsx [hunk 3]
```diff
@@ -21,7 +22,9 @@ const Abacard = loadable(() => import('./Abacard'));
 
 const EventAdministrateIndex = () => {
   const { eventId } = useParams<{ eventId: string }>();
-  const event = useAppSelector((state) => selectEventById(state, { eventId }));
+  const event = useAppSelector((state) =>
+    eventId ? selectEventById(state, { eventId }) : undefined,
```
Comment (Bestem0r) [inline app/routes/events/components/EventAdministrate/index.tsx:26]: Same here?

**Review by Bestem0r — APPROVED** [review]
Great work!

**Review by falbru — APPROVED** [review]

**Pull request merged**